app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserViewModel.kt

```diff
@@ -65,9 +65,11 @@ class OrdersUnderUserViewModel(
     private fun eventsUnderUser(eventIds: List<Long>, showExpired: Boolean) {
         compositeDisposable += eventService.getEventsUnderUser(eventIds)
             .withDefaultSchedulers()
+            .distinctUntilChanged()
             .doFinally {
                 mutableShowShimmerResults.value = false
             }.subscribe({
+                mutableShowShimmerResults.value = false
```

Comment:
WHY?

Reply:
`Calls the specified action after this Flowable signals onError or onCompleted or gets canceled by the downstream.`

Means it'll keep showing the shimmer even after the success.